### ISSUE TITLE
Bug - Nozzle sinks into model when Octolapse move occurs

### ISSUE THREAD

**Issue by mneumark** [Kragrathea/OctoPrint-PrettyGCode#42]
I have Octolapse running which means at some point once per layer it moves the extruder over to left-center (in my case). when this happens in PrettyGCode it just sinks into the model and then pops back when the move is over.

**Comment by Kragrathea** [comment]
Apparently Octolapse inserts a GCode command to move the print head to Left Center and that command isn't in exactly the format I expect. If you can figure out exactly what that command is I can try to fix it. Otherwise I will have to come back to this when I can find time to get Octolapse working on my setup. I use my camera for something else.

**Comment by energiebrand** [comment]
Yesterday I did my first print with "Pretty GCode". I noticed that a similar behaviour occurs during the last and final layer. The nozzle suddenly droped down to layer 0 and finished there. The gcode was generated by CURA 4.6.1. Unfortunately I did not make a screen shot, but since there a some other GUI problems when using different browsers I will soon open a new issue and add here a screenshot of the dropped nozzle. By the way, I do not use Octolapse!
